### PR TITLE
add note about DO bindings in the getPlatformProxy docs

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

File: content/workers/wrangler/api.md [hunk 1]
```diff
@@ -343,6 +343,14 @@ The bindings supported by `getPlatformProxy` are:
 
  * [Durable Object bindings](/workers/configuration/bindings/#durable-object-bindings)
 
+    * To use a Durable Object binding with `getPlatformProxy`, you need to always specify a `script_name` for it
```
Comment (irvinebroque) [inline content/workers/wrangler/api.md]:
```suggestion
    * To use a Durable Object binding with `getPlatformProxy`, you need to always specify a `script_name` for it
```

Someone building an application directly on Workers doesn't normally interact with or know about `getPlatformProxy` — is what you really mean here something like "to use Durable Objects in a Pages project"?

Comment (irvinebroque) [reply content/workers/wrangler/api.md]: Oh nvm I get it now where these docs are.

**Review by irvinebroque — APPROVED** [review]

File: content/workers/wrangler/api.md [hunk 2]
```diff
@@ -343,6 +343,10 @@ The bindings supported by `getPlatformProxy` are:
 
  * [Durable Object bindings](/workers/configuration/bindings/#durable-object-bindings)
 
+    * To use a Durable Object binding with `getPlatformProxy`, you need to always [specify a `script_name`](/workers/wrangler/configuration/#durable-objects) and have the target Worker run in a separate terminal via [`wrangler dev`](/workers/wrangler/commands/#dev).
+
+      For example, in your Worker project's `wrangler.toml` file, you have a Durable Object with certain `class_name` and `script_name` set to `"my-worker"`. The same exact `class_name` is also used by a Durable Object binding in your `my-worker` Worker, and you are running the Worker via `wrangler dev`. Calling `getPlatformProxy` will then successfully generate a proxy to the Durable Object defined in the `my-worker` Worker.
```
Comment (deadlypants1973) [inline content/workers/wrangler/api.md:348]: Shouldn't `class_name` be set to the name of the class, not the Worker? I am confused about this example.

**Review by deadlypants1973 — COMMENTED** [review]
Example unclear

File: content/workers/wrangler/api.md (hunk 2 above)
Comment (deadlypants1973) [inline content/workers/wrangler/api.md]:
```suggestion
      For example, in your Worker project's `wrangler.toml` file, you have a Durable Object with `script_name` set to `"my-worker"` and a certain `class_name`. The same exact `class_name` is also used by a Durable Object binding in your `my-worker` Worker, and you are running the Worker via `wrangler dev`. Calling `getPlatformProxy` will then successfully generate a proxy to the Durable Object defined in the `my-worker` Worker.
```

Comment (deadlypants1973) [reply content/workers/wrangler/api.md]: This example needs some clarity as I am not sure by reading how many Workers are involved in this example

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

File: content/workers/wrangler/api.md (hunk 2 above)
Comment (dario-piotrowicz) [reply content/workers/wrangler/api.md]: I gave it a shot, please have another look 🙏

**Pull request closed without merging**